docs/MO_DG/prepare_model/convert_model/Convert_Model_From_TensorFlow.md

```diff
@@ -108,12 +108,27 @@ If a model contains operations currently unsupported by OpenVINO™,
 prune these operations by explicit specification of input nodes using the ``--input`` or ``--output``
 options. To determine custom input nodes, visualize a model graph in the TensorBoard.
 
-To generate TensorBoard logs of the graph, use the Model Optimizer ``--tensorboard_logs`` command-line
-option.
-
 TensorFlow 2 SavedModel format has a specific graph structure due to eager execution. In case of
 pruning, find custom input nodes in the ``StatefulPartitionedCall/*`` subgraph.
 
+Since 2023.0 release, direct pruning of models in SavedModel format is not supported.
+Prior to the pruning, it is essential to freeze the model. The model freezing can be achieved using the following code snippet:
```

Comment:
```suggestion
Since the 2023.0 release, direct pruning of models in SavedModel format is not supported.
It is essential to freeze the model before pruning. Use the following code snippet for model freezing: 
```